concepts/ownership/about.md

```diff
@@ -1 +1,118 @@
 # Ownership
+In Cairo, the concept of ownership is attached to variables rather than the values they hold.
+This means that while multiple variables can refer to the same value, ownership is concerned with the variables themselves.
+Values in Cairo are immutable, meaning once a value is created, it cannot be changed.
+This immutability ensures that even if many variables refer to the same value, the value itself remains constant and unaltered.
+
+ ## The ownership rules.
+  _make sure to keep these rules in mind as you work with Cairo_
+```
+- Each variable in Cairo has an owner.
+- There can only be one owner at a time.
+- When the owner goes out of scope, the variable is destroyed.
+```
+### Variable scope
+Technically a scope is the range within a program in which a variable is valid
+
+```rust
+fn main(){
+     let x = 3;
+}
+   
+```
+the `x` is an integer which is valid at the point in which it is declared up to the end of the given scope, in this case `main()` function.
+
+```rust
+fn main(){ 
+                      //x is not valid here; it has not been declared
+     let x = 3;       //x is valid from this point forward ; it is been declared
+// do stuffs with x   // x is also valid here; it is still in scope
+
+} // this scope ended here and x goes out of scope and no longer valid.
+   
+```
+Note: 
+ - When `x` comes into scope, it is valid.
+ - It remains valid until it goes out of scope.
+
+
+ ### Moving Vaues
+A move occurs when a value is passed to another function, rendering the original variable that referred to the value useless. This happens because moving a value causes the original variable to be destroyed, making it unusable.
+
+complex data types are the types of data that is moved when passed into a function and array serves as a good example;
```

Comment:
I think it's best we reuse word-for-word the content from [The Cairo Book > Moving values](https://book.cairo-lang.org/ch04-01-what-is-ownership.html#moving-values) - the explanation and the example are very simple, so we should reuse.

Update the rest of this section with The Book's content, starting from:
> Arrays are an example of a complex type that is moved when passing it to another function. So how does the type system ensure that the Cairo program never tries to write to the same memory cell twice? Consider the following code, where we try to remove the front of the array twice:
...

Then insert The Book's example and the paragraph after it. 
You can include the inline comments explaining each line, I like that.

You don't need to add the compilation result section from The Book though, that would be redundant.